src/main/java/ai/lucidtech/las/sdk/CreateWorkflowOptions.java

```diff
@@ -0,0 +1,34 @@
+package ai.lucidtech.las.sdk;
+
+import org.json.JSONObject;
+
+
+public class CreateWorkflowOptions extends NameAndDescriptionOptions<CreateWorkflowOptions> {
+    private JSONObject errorConfig;
+
+
+    public CreateWorkflowOptions(){
+        this.errorConfig = null;
+    }
+
+    public CreateWorkflowOptions(JSONObject errorConfig){
+        this.errorConfig = errorConfig;
+    }
```

Comment:
Why does this class have a constructor option for fields why other builders do not?

Reply:
I added it when it made sense. If only one argument will be added, or the arguments are of different types. But for two optional strings for instance it just seems confusing.

Reply:
I might as well remove it if you think it doesn't provide any help. 
But to me it seems easier to do 
```
new Options(input);
// rather than
new Options().setInput(input);
```